README.md

```diff
@@ -0,0 +1,713 @@
+# node-diacritics-transliterator
+
+> Diacritic transliteration tools using diacritics.io API
+
+# Installation
+
+```console
+$ npm install diacritics-transliterator
+```
+
+# Usage
+
+```js
+var diacritics = require("diacritics-transliterator");
+diacritics.setVersion("v1");
```

Comment:
I thought `diacritics.setVersion("v1");` is optional, so we don't need to have this explicit named in the minimal example.